server/app/services/applicant/question/AddressQuestion.java

```diff
@@ -212,4 +212,9 @@ public ImmutableList<Path> getAllPaths() {
     return ImmutableList.of(
         getStreetPath(), getLine2Path(), getCityPath(), getStatePath(), getZipPath());
   }
+
+  public Optional<String> getDefaultState() {
+    AddressQuestionDefinition definition = getQuestionDefinition();
+    return definition.getDefaultState();
```

Comment:
Personally I don't think that method is useful. Callers can always get question definition themselves and retrieve default state from there. And default state is semantically part of QuestionDefinition and not Question itself (values that user input).